app/src/main/java/io/seqera/wave/cli/App.java

```diff
@@ -241,6 +241,8 @@ else if( result.isVersionHelpRequested() ) {
 
             app.setLogLevel();
             app.defaultArgs();
+            // validate the command line args
+            app.validateArgs();
```

Comment:
Not sure it's needed

Reply:
inspect is running before this validation so either i can validate it in inspect method or move validate method before inspect method